src/Processors/QueryPlan/ReadFromMergeTree.cpp

```diff
@@ -2309,10 +2309,19 @@ void ReadFromMergeTree::initializePipeline(QueryPipelineBuilder & pipeline, cons
         Names partition_names;
         for (const auto & part : result.parts_with_ranges)
         {
-            partition_names.emplace_back(
-                fmt::format("{}.{}", data.getStorageID().getFullNameNotQuoted(), part.data_part->info.getPartitionId()));
+            partition_names.emplace_back(part.data_part->info.getPartitionId());
```

Comment:
In l. 2306, please change to `Add partition info and skipping index info to QueryAccessInfo.`.